cmd/restic/cmd_ls.go

```diff
@@ -84,7 +84,7 @@ type lsNode struct {
 }
 
 func runLs(opts LsOptions, gopts GlobalOptions, args []string) error {
-	if len(args) == 0 && opts.Host == "" && len(opts.Tags) == 0 && len(opts.Paths) == 0 {
+	if len(args) == 0 && opts.Hosts == nil && len(opts.Tags) == 0 && len(opts.Paths) == 0 {
```

Comment:
Use `len(opts.Hosts) == 0` instead of checking equality to `nil`, that's more robust.

Reply:
is changed.